posthog/models/feature_flag.py

```diff
@@ -640,31 +682,58 @@ def get_all_feature_flags(
     hash_key_override: Optional[str] = None,
     property_value_overrides: Dict[str, Union[str, int]] = {},
     group_property_value_overrides: Dict[str, Dict[str, Union[str, int]]] = {},
-) -> Tuple[Dict[str, Union[str, bool]], Dict[str, dict], Dict[str, object]]:
-
-    all_feature_flags = FeatureFlag.objects.filter(team_id=team_id, active=True, deleted=False).only(
-        "id", "team_id", "filters", "key", "rollout_percentage", "ensure_experience_continuity"
-    )
+) -> Tuple[Dict[str, Union[str, bool]], Dict[str, dict], Dict[str, object], bool]:
+
+    all_feature_flags = get_feature_flags_for_team_in_cache(team_id)
+    if all_feature_flags is None:
+        all_feature_flags = list(
+            FeatureFlag.objects.filter(team_id=team_id, active=True, deleted=False).only(
```

Comment:
Why not just call `set_feature_flags_for_team_in_cache` without `all_feature_flags` and skip this manual call on the queryset

Reply:
because then we'll have to call `get` again to get the flags to use in the following functions

Reply:
Could have "set" also return flags? My motivation for that question was just some way to make sure filters stay in line if it ever needs adjustment because right now `FeatureFlag.objects.filter(team_id=team_id, active=True, deleted=False)` is ad-hoc in two places

Reply:
ah, I see. Yep makes sense